### DISCUSSION TITLE
add [@types/react-datepicker] prop renderQuarterContent

### DISCUSSION THREAD

**Discussion by exsens** [DefinitelyTyped/DefinitelyTyped#68661] [Issues with a @types package]
Need to add typing for the renderQuarterContent prop:
renderQuarterContent: (quarter: number, shortQuarterText: string) => React.ReactNode;
Provide a URL to documentation or source code which provides context for the suggested changes: [react-datepicker documentation](https://reactdatepicker.com/#example-custom-quarter)

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in PR: #68682
